code/src/App.js

```diff
@@ -1,12 +1,37 @@
 import React from 'react'
 import data from './data.json'
+import './app.css'
+import { Artist } from "./components/Artist";
+import { Album } from "./components/Album";
 
-console.log(data)
+const renderAlbum = album => {
+  const { id, images, name, external_urls } = album;
+  return (
+    <div key={id} className="artist-container">
+      <Album 
+      image={images[1].url} 
+      name={name} 
+      url={external_urls.spotify} 
+      />
+      {album.artists.map(renderArtist)}
+    </div>
+  );
+};
```

Comment:
There's nothing wrong with this approach (making a function to render an element and then invoking it in a map), but in larger apps this can make things harder to follow because you have to jump around a lot to read whats going on. I'd push for doing the .map within the returning JSX as much as possible, until you hit a point where pulling it up and out is warranted.